hazelcast/src/main/java/com/hazelcast/spi/impl/eventservice/impl/Registration.java

```diff
@@ -29,14 +30,16 @@
 import java.io.IOException;
 import java.util.UUID;
 
-public class Registration implements EventRegistration {
+import static com.hazelcast.internal.cluster.Versions.V5_3;
+
+public class Registration implements EventRegistration, Versioned {
 
     private UUID id;
     private String serviceName;
     private String topic;
     private EventFilter filter;
     private Address subscriber;
-    private transient boolean localOnly;
+    private boolean localOnly;
```

Comment:
Subscriber address should be sufficient to check locality. We can keep localOnly` as `transient`.